### PR TITLE
bug(replay-button): fix replay button

### LINKED ISSUE #103: fix replay button on the practice game mode
**Is your feature request related to a problem? Please describe.**
When in the practice mode the replay button starts the player in the main game mode

**Describe the solution you'd like**
For the game the player is practicing to be replayed

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Have the replay button work differently depending on the game mode
- If from the main game mode the replay button will reshuffle all the games and reset your finished games position to none
- if from the practice game mode the game will just replay 

## Purpose
To have a working replay button so a player can keep playing a game without having to go back to the main page

Closes #103

### CODE REVIEW

File: components/MenuModal.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ const MenuModal = () => {
     const route = useRoute();
 
     const fromGameSelect = route.params.fromGameSelect
+    const chosenGameKey = route.params.chosenGame
+
+    console.log("game array length", gamesArray.length)
+    console.log("game index plus 1", gameIndex + 1)
```
Comment (Lap343) [inline components/MenuModal.js:29]: oh no

**Review by Gjhnsn — DISMISSED** [review]
looks good

**Review by dannysanchez559 — APPROVED** [review]
nice work

File: components/MenuModal.js (hunk 1 above)
Comment (BlackCubes) [inline components/MenuModal.js]: Wat's DIS?????????

**Review by BlackCubes — APPROVED** [review]
Console.log!!!! But gewd job!

File: components/MenuModal.js (hunk 1 above)
Comment (Lap343) [reply components/MenuModal.js]: I'm magic! It's gone now

**Review by connielion — APPROVED** [review]
Noice!

**Review by JorgeLVilla — APPROVED** [review]

**Pull request merged**